src/NServiceBus.RavenDB/Timeouts/TimeoutPersister.cs

```diff
@@ -148,5 +149,32 @@ IRavenQueryable<Timeout> GetChunkQuery(IDocumentSession session)
                 return chunk;
             }
         }
+
+        public CoreTimeoutData Peek(string timeoutId)
+        {
+            using (var session = DocumentStore.OpenSession())
+            {
+                var timeoutData = session.Load<Timeout>(timeoutId);
+                if (timeoutData != null)
+                {
+                    return timeoutData.ToCoreTimeoutData();
+                }
+
+                return null;
+            }
+        }
+
+        public bool TryRemove(string timeoutId)
+        {
+            try
+            {
+                CoreTimeoutData timeoutData;
+                return TryRemove(timeoutId, out timeoutData);
```

Comment:
Remind me: Why don't we have that try catch for the concurrency ex? (in the original try remove)